packages/adaptive-ui/src/core/color/palette-okhsl.ts

```diff
@@ -17,39 +52,46 @@ const threeSteps = (1 / stepCount) * 3;
  * @public
  */
 export class PaletteOkhsl extends BasePalette<Swatch> {
-    public static from(source: Color | string): PaletteOkhsl {
+    public static from(source: Color | string, options?: Partial<PaletteOkhslOptions>): PaletteOkhsl {
         const color = source instanceof Color ? source : Color.parse(source);
         if (!color) {
             throw new Error(`Unable to parse source: ${source}`);
         }
 
+        const opts = options === void 0 || null ? defaultPaletteOkhslOptions : { ...defaultPaletteOkhslOptions, ...options };
```

Comment:
`void 0 || null` will only check `null` because `void 0` is falsey. If you want to check for both you'll need to separate the expressions (options === void 0 || options === null)`